conf.go

```diff
@@ -161,15 +162,44 @@ func String(v interface{}) (string, error) {
 	}
 
 	var s strings.Builder
+	var tempval string
 	for i, fld := range fields {
 		if !fld.Options.Noprint {
 			s.WriteString(flagUsage(fld))
 			s.WriteString("=")
-			s.WriteString(fmt.Sprintf("%v", fld.Field.Interface()))
-			if i < len(fields)-1 {
-				s.WriteString("\n")
+			if fld.Options.Mask {
```

Comment:
I can tell you this already. Let's leave noprint alone and add this as a new flag and a new code path inside the loop.

Not a fan of else clauses, haven't looked enough to know if there is no way around it.